src/postgres/getCatalog.js

```diff
@@ -75,7 +75,22 @@ const addTables = (client, catalog) =>
     select
       n.nspname as "schemaName",
       c.relname as "name",
-      d.description as "description"
+      d.description as "description",
+      case when
+           (pg_catalog.pg_relation_is_updatable(c.oid, true)::bit(8) & B'00010000') = B'00010000'
```

Comment:
Yep, this seems to be the most comprehensive. Good job 😊

Would you mind throwing those links in a comment in the code?